### PR TITLE
Adding AU Payroll and Bankfeeds

### PR DESCRIPTION
**Pull request by SerKnight**
Adds in all functionality in the 4.6.0 release of Xero Node https://github.com/XeroAPI/xero-node/releases

Also a lot of work to make the readme more clear for people getting started.

Included session persistence so that every typescript rebuild doesn't clobber the active session which includes your tokenSet and other information.

------------

Largest changes are adding in the functionality of 2 XeroAPI sets:
* [Overview - Payroll - Australia | Xero Developer](https://developer.xero.com/documentation/payroll-api/overview)
* [Overview - Bank Feeds | Xero Developer](https://developer.xero.com/documentation/bank-feeds-api/overview)

### CODE REVIEW

**Comment by SerKnight** [comment]
Work IN Progress do not merge

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "module-alias": "^2.2.2",
     "nodemon": "^1.19.4",
     "path": "^0.12.7",
-    "xero-node": "^4.5.1"
+    "session-file-store": "^1.4.0",
+    "xero-node": "/Users/chris.knight/code/sdks/xero-node/xero-node-4.6.0.tgz"
```
Comment (RettBehrens) [inline package.json]: you'll want to point to the library instead of local before merging

File: src/app.ts [hunk 2]
```diff
@@ -56,21 +56,27 @@ import {
...
+  Employee,
 } from "xero-node";
 import Helper from "./helper";
 import jwtDecode from 'jwt-decode';
 import { Asset } from "xero-node/dist/gen/model/assets/asset";
 import { AssetStatus, AssetStatusQueryParam } from "xero-node/dist/gen/model/assets/models";
 import { Project, ProjectCreateOrUpdate, ProjectPatch, ProjectStatus, TimeEntry, TimeEntryCreateOrUpdate } from 'xero-node/dist/gen/model/projects/models';
+import { Employee as AUPayrollEmployee, HomeAddress, State, EmployeeStatus, EarningsType } from 'xero-node/dist/gen/model/payroll-au/models';
+import { FeedConnections, FeedConnection, CountryCode, Statements, Statement, CreditDebitIndicator, CurrencyCode as BankfeedsCurrencyCode } from 'xero-node/dist/gen/model/bankfeeds/models';
+import { disconnect } from "cluster";
```
Comment (RettBehrens) [inline src/app.ts]: this is an unused import

**Review by RettBehrens — COMMENTED** [review]
see comments before merging

**Pull request merged**